ui/desktop/index.html

```diff
@@ -8,20 +8,42 @@
       // Initialize theme before any content loads
       (function() {
         function initializeTheme() {
-          const useSystemTheme = localStorage.getItem('use_system_theme') === 'true';
-          const systemPrefersDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
-          const savedTheme = localStorage.getItem('theme');
-          const isDark = useSystemTheme ? systemPrefersDark : (savedTheme ? savedTheme === 'dark' : systemPrefersDark);
-          
-          if (isDark) {
-            document.documentElement.classList.add('dark');
-          } else {
-            document.documentElement.classList.remove('dark');
+          try {
+            if (typeof Storage !== 'undefined' && window.localStorage) {
```

Comment:
when would storage be undefined?

Reply:
would be in a pretty bad state if we hit that in the electron app though wouldn't it?

Reply:
Yeah I am also a bit confused on this point. Wouldn't electron bundle the APIs being used, so it would either be defined or not in a given implementation?

I'm not against checking, but just wondering if it could ever catch.